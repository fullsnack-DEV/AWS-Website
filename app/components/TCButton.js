import React from 'react';
import {
  StyleSheet,

  Text,
  TouchableOpacity,

} from 'react-native';
import constants from '../config/constants';

const {
  colors, fonts,
} = constants;

function TCButton({
  title,
  onPress,
  extraStyle,
  textColor = { color: colors.themeColor },
}) {
  return (
      <TouchableOpacity onPress={ onPress } style={ [styles.allButton, extraStyle] }>
          <Text style={ [styles.signUpText, textColor] }>{title}</Text>
      </TouchableOpacity>
  );
}

const styles = StyleSheet.create({
  allButton: {
    alignContent: 'center',
    backgroundColor: colors.whiteColor,
    borderRadius: 40,
    height: 50,
    marginLeft: '7%',
    marginRight: '5%',
    shadowColor: colors.googleColor,

    shadowOffset: { width: 0, height: 2 },

    shadowOpacity: 0.5,
    shadowRadius: 5,
    width: '86%',
  },

  signUpText: {
    fontFamily: fonts.RBold,
    fontSize: 17,
    height: 50,
    padding: 12,
    textAlign: 'center',
  },
});

export default TCButton;
