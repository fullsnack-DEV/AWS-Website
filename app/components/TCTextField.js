import React from 'react';
import {
  StyleSheet,

  TextInput,
  View,
} from 'react-native';

import colors from '../Constants/Colors'
import fonts from '../Constants/Fonts'

function TCTextField({
  placeholder,
  placeholderTextColor = colors.userPostTimeColor,
  secureText = false,
  keyboardType = 'default',
  style,
  textStyle,
  ...otherProps
}) {
  return (
    <View style={[styles.textContainer, style]}>
      <TextInput
        style={ [styles.textInput, textStyle] }
        placeholder={ placeholder }
        placeholderTextColor={ placeholderTextColor }
        secureTextEntry={ secureText }
        keyboardType={ keyboardType }
        { ...otherProps }
    />
    </View>
  );
}

const styles = StyleSheet.create({
  textContainer: {
    height: 40,
    marginHorizontal: 15,
    backgroundColor: colors.offwhite,
    borderRadius: 5,
    shadowColor: colors.blackColor,
    shadowOffset: { width: 0, height: 1 },
    shadowOpacity: 0.16,
    shadowRadius: 1,
    elevation: 3,

  },
  textInput: {
    height: '100%',
    fontFamily: fonts.RRegular,
    fontSize: 16,
    paddingHorizontal: 10,
    color: colors.lightBlackColor,
  },
});

export default TCTextField;

/*
keyboardType :
The following values work across platforms:

default
number-pad
decimal-pad
numeric
email-address
phone-pad
*/
