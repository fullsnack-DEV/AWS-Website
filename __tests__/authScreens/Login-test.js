/* eslint-disable no-undef */
import React from 'react';
import {render, fireEvent} from '@testing-library/react-native';
import axios from 'axios';

import LoginScreen from '../../app/screens/authScreens/LoginScreen';
import TCTextField from '../../app/components/TCTextField';
import TCButton from '../../app/components/TCButton';

const email = 'test@example.com';
const password = '123456';

afterEach(() => {
  jest.clearAllMocks();
});

describe('Login  Screen Test ', () => {
  jest.mock('axios');
  axios.get = jest.fn();
  const navigate = jest.fn();
  const setOptions = jest.fn();
  const goBack = jest.fn();

  it('should match the snapshot', () => {
    const rendered = render(
      <LoginScreen
        navigation={{goBack, navigate, setOptions}}
        testID={'password-input'}
      />,
    ).toJSON();
    expect(rendered).toMatchSnapshot();
  });

  it('email not blank ', () => {
    const {getByTestId} = render(
      <TCTextField testID="email-input" value={email} />,
    );
    const wrapperComponent = getByTestId('email-input');
    expect(wrapperComponent.props.value).not.toBe('');
  });

  //   test('password not blank ', () => {
  //     const {getByTestId} = render(
  //       <LoginScreen navigation={{goBack, navigate, setOptions}} />,
  //     );
  //     const input = getByTestId('password-input');
  //     expect(input.props.value).not.toBe('');
  //   });

  it('Login mock api call', async () => {
    const handleClick = jest.fn();
    const {getByTestId} = render(
      <TCButton testID="login-button" onPress={handleClick} />,
    );
    const loginButton = getByTestId('login-button');
    fireEvent.press(loginButton);
    expect(handleClick).toHaveBeenCalledTimes(1);
    axios.get
      .mockImplementationOnce(() =>
        Promise.resolve({
          status: 200,
          data: {
            email,
            password,
          },
        }),
      )
      .mockImplementationOnce(() =>
        Promise.resolve({
          status: 400,
          data: null,
        }),
      );
  });
});
