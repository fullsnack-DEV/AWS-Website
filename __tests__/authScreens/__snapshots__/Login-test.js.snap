// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login  Screen Test  should match the snapshot 1`] = `
<BVLinearGradient
  colors={
    Array [
      4294949633,
      4294924288,
    ]
  }
  endPoint={
    Object {
      "x": 0.5,
      "y": 1,
    }
  }
  locations={null}
  startPoint={
    Object {
      "x": 0.5,
      "y": 0,
    }
  }
  style={
    Object {
      "flex": 1,
      "paddingVertical": 25,
    }
  }
>
  <View
    style={
      Array [
        Object {
          "overflow": "hidden",
        },
        Object {
          "height": 1334,
          "position": "absolute",
          "width": 750,
        },
      ]
    }
  >
    <FastImageView
      resizeMode="stretch"
      source={
        Object {
          "default": Object {
            "Invoicing": [Circular],
            "Members": [Circular],
            "Payment_method": [Circular],
            "Payout_method": [Circular],
            "SettingPrivacy": [Circular],
            "Transations": [Circular],
            "accountMyClubs": [Circular],
            "accountMyLeagues": [Circular],
            "accountMyRefereeing": [Circular],
            "accountMySchedule": [Circular],
            "accountMyScoreKeeping": [Circular],
            "accountMySports": [Circular],
            "accountMyTeams": [Circular],
            "accountPaymentPayout": [Circular],
            "accountSettingPrivacy": [Circular],
            "addEvent": [Circular],
            "addMessageChat": [Circular],
            "addRole": [Circular],
            "addSport": [Circular],
            "addSportList": [Circular],
            "addStatus": [Circular],
            "appleLogo": [Circular],
            "archerySport": [Circular],
            "arrowDown": [Circular],
            "arrowGT": [Circular],
            "arrowGraterthan": [Circular],
            "assistBy": [Circular],
            "assistReferee": [Circular],
            "assistsImage": [Circular],
            "assistsSelected": [Circular],
            "availableChallenge": [Circular],
            "backArrow": [Circular],
            "backetballSport": [Circular],
            "backgroudPlaceholder": [Circular],
            "backgroundGrayPlceholder": [Circular],
            "backgroundPlaceholder": [Circular],
            "badmintonLoader": [Circular],
            "bandySport": [Circular],
            "baseballLoader": [Circular],
            "baseballSport": [Circular],
            "basketballLoader": [Circular],
            "bgImage": [Circular],
            "blankRatingStar": [Circular],
            "blockedChallenge": [Circular],
            "blueGradient": [Circular],
            "blueRatingStar": [Circular],
            "calNextArrow": [Circular],
            "calPrevArrow": [Circular],
            "cancelImage": [Circular],
            "cancelWhite": [Circular],
            "certiImagePick": [Circular],
            "certificateUpload": [Circular],
            "chain": [Circular],
            "challengeSentPlane": [Circular],
            "challengeeBadge": [Circular],
            "challengeeCup": [Circular],
            "challengerBadge": [Circular],
            "challengerFlag": [Circular],
            "challengerInvitePlane": [Circular],
            "chat3Dot": [Circular],
            "chatPlus": [Circular],
            "chatSearch": [Circular],
            "check": [Circular],
            "checkEditor": [Circular],
            "checkGreen": [Circular],
            "checkGreenBG": [Circular],
            "checkRoundOrange": [Circular],
            "checkWhite": [Circular],
            "checkWhiteLanguage": [Circular],
            "choosePic": [Circular],
            "closeSearch": [Circular],
            "clubC": [Circular],
            "clubCover": [Circular],
            "clubLable": [Circular],
            "clubPatch": [Circular],
            "clubPlaceholder": [Circular],
            "clubPlaceholderSmall": [Circular],
            "clubSmallIcon": [Circular],
            "clubSqure": [Circular],
            "club_ph": [Circular],
            "commentDeleteIcon": [Circular],
            "commentImage": [Circular],
            "commentReport": [Circular],
            "commentReportIcon": [Circular],
            "createClub": [Circular],
            "createEventImage": [Circular],
            "createLeague": [Circular],
            "createTeam": [Circular],
            "crossImage": [Circular],
            "curruentTime": [Circular],
            "declineChallenge": [Circular],
            "deleteIcon": [Circular],
            "deleteRecentGoal": [Circular],
            "dotImage": [Circular],
            "dottedLine": [Circular],
            "doubleTeamCreated": [Circular],
            "downArrow": [Circular],
            "dropDownArrow": [Circular],
            "dropDownArrow2": [Circular],
            "dummyPhoto": [Circular],
            "editButton": [Circular],
            "editPencil": [Circular],
            "editSection": [Circular],
            "email": [Circular],
            "emailSendIconBG": [Circular],
            "emailSent1": [Circular],
            "emailSent2": [Circular],
            "emailSent3": [Circular],
            "emailSent4": [Circular],
            "emailSent5": [Circular],
            "entityCreatedBG": [Circular],
            "feedComment": [Circular],
            "feedLike": [Circular],
            "feedPlusIcon": [Circular],
            "feedShare": [Circular],
            "feedViewCommentButton": [Circular],
            "feedViewLikeButton": [Circular],
            "feedViewShareButton": [Circular],
            "feedViewUnLike": [Circular],
            "football2Loader": [Circular],
            "footballLoader": [Circular],
            "footballSport": [Circular],
            "galleryVideoIcon": [Circular],
            "gameDetail": [Circular],
            "gameGoal": [Circular],
            "gameIn": [Circular],
            "gameOrangePlus": [Circular],
            "gameOut": [Circular],
            "gameOwnGoal": [Circular],
            "gamePause": [Circular],
            "gamePlus": [Circular],
            "gameRC": [Circular],
            "gameRecord": [Circular],
            "gameSimple": [Circular],
            "gameStart": [Circular],
            "gameYC": [Circular],
            "gamesImage": [Circular],
            "gamesSelected": [Circular],
            "goalsImage": [Circular],
            "goalsSelected": [Circular],
            "grayListIcon": [Circular],
            "greenGradient": [Circular],
            "greenQBGroup": [Circular],
            "greenQBUser": [Circular],
            "greenRatingStar": [Circular],
            "groupIcon": [Circular],
            "groupUsers": [Circular],
            "hidePassword": [Circular],
            "homeSetting": [Circular],
            "home_gps": [Circular],
            "home_search": [Circular],
            "home_setting": [Circular],
            "horizontal3Dot": [Circular],
            "imageLoadingGIF": [Circular],
            "imageUploadingGIF": [Circular],
            "infoToolTipIcon": [Circular],
            "landscapeBottomImage": [Circular],
            "landscapeTopImage": [Circular],
            "landscapeVideoImage": [Circular],
            "leagueCover": [Circular],
            "leagueDemo": [Circular],
            "leagueL": [Circular],
            "leaguePatch": [Circular],
            "leaguePlaceholder": [Circular],
            "leagueSmallIcon": [Circular],
            "leave_chat_room": [Circular],
            "likeImage": [Circular],
            "likePlay": [Circular],
            "linkCopied": [Circular],
            "localHomeFilter": [Circular],
            "localhomeOverlay": [Circular],
            "lock": [Circular],
            "loginBg": [Circular],
            "logoutIcon": [Circular],
            "manageChallengeIcon": [Circular],
            "menu": [Circular],
            "menuClose": [Circular],
            "menuGray": [Circular],
            "menuOrange": [Circular],
            "messageCamera": [Circular],
            "messageCheckboxBorder": [Circular],
            "messageIcon": [Circular],
            "messageSearchButton": [Circular],
            "messageSearchButton2": [Circular],
            "moveFlottyButton": [Circular],
            "moveIcon": [Circular],
            "multipleImagesIcon": [Circular],
            "mute": [Circular],
            "myClubs": [Circular],
            "myLeagues": [Circular],
            "myRefereeing": [Circular],
            "mySchedule": [Circular],
            "myScoreKeeping": [Circular],
            "mySports": [Circular],
            "myTeams": [Circular],
            "navigationBack": [Circular],
            "newFeed": [Circular],
            "nextArrow": [Circular],
            "noSpecicPlayer": [Circular],
            "notificationBell": [Circular],
            "orangeCheckBox": [Circular],
            "orangeGradient": [Circular],
            "orangeGradiet": [Circular],
            "orangeLayer": [Circular],
            "orangeListIcon": [Circular],
            "orangeStar": [Circular],
            "paymentPayout": [Circular],
            "pickImage": [Circular],
            "playerIcon": [Circular],
            "playerSucessBG": [Circular],
            "plus": [Circular],
            "plusInvoice": [Circular],
            "plus_round_orange": [Circular],
            "popupUpArrow": [Circular],
            "portraitVideoImage": [Circular],
            "profileLevel": [Circular],
            "profilePlaceHolder": [Circular],
            "radioCheckGreenBG": [Circular],
            "radioCheckYellow": [Circular],
            "radioRoundOrange": [Circular],
            "radioSelect": [Circular],
            "radioSelectGreen": [Circular],
            "radioSelectYellow": [Circular],
            "radioUnSelectGreen": [Circular],
            "radioUnselect": [Circular],
            "recentMatch": [Circular],
            "refIcon": [Circular],
            "referePH": [Circular],
            "refereeIcon": [Circular],
            "refereeSucessBG": [Circular],
            "refereesInImage": [Circular],
            "registerReferee": [Circular],
            "registerScorekeeper": [Circular],
            "removeSportList": [Circular],
            "reqIcon": [Circular],
            "reqeIcon": [Circular],
            "requestIn": [Circular],
            "requestOut": [Circular],
            "requestSentImage": [Circular],
            "resetColor": [Circular],
            "roundArrow": [Circular],
            "savedIcon": [Circular],
            "scheduleGray": [Circular],
            "scheduleOrange": [Circular],
            "scheduleThreeDot": [Circular],
            "scorekeeperIcon": [Circular],
            "scorekeeperPH": [Circular],
            "searchLocation": [Circular],
            "searchUser": [Circular],
            "selectAddMessageButton": [Circular],
            "selectMessageSearchButton": [Circular],
            "sendButton": [Circular],
            "settingInvoice": [Circular],
            "settingSport": [Circular],
            "shareImage": [Circular],
            "shortCard": [Circular],
            "shortComment": [Circular],
            "shortDisLike": [Circular],
            "shortLike": [Circular],
            "shortShare": [Circular],
            "showPassword": [Circular],
            "signUpFb": [Circular],
            "signUpGoogle": [Circular],
            "smallRight": [Circular],
            "soccerBackground": [Circular],
            "soccerIcon": [Circular],
            "soccerImage": [Circular],
            "soccerLoader": [Circular],
            "swapTeam": [Circular],
            "switchAccount": [Circular],
            "tabFeed": [Circular],
            "tabSelectedFeed": [Circular],
            "tab_account": [Circular],
            "tab_account_group": [Circular],
            "tab_account_group_selected": [Circular],
            "tab_account_selected": [Circular],
            "tab_home": [Circular],
            "tab_home_selected": [Circular],
            "tab_message": [Circular],
            "tab_message_selected": [Circular],
            "tab_notification": [Circular],
            "tab_notification_selected": [Circular],
            "tab_reservation": [Circular],
            "tab_reservation_selected": [Circular],
            "tagDivider": [Circular],
            "tagGreenImage": [Circular],
            "tagImage": [Circular],
            "tc_message_top_icon": [Circular],
            "teamCover": [Circular],
            "teamGreenPH": [Circular],
            "teamPH": [Circular],
            "teamPatch": [Circular],
            "teamPlaceholder": [Circular],
            "teamPlaceholderSmall": [Circular],
            "teamSmallIcon": [Circular],
            "teamSqure": [Circular],
            "teamT": [Circular],
            "team_ph": [Circular],
            "tennisAce": [Circular],
            "tennisArrowLeft": [Circular],
            "tennisArrowRight": [Circular],
            "tennisBackground": [Circular],
            "tennisFault": [Circular],
            "tennisFootFault": [Circular],
            "tennisGeneral": [Circular],
            "tennisIcon": [Circular],
            "tennisLet": [Circular],
            "tennisScore": [Circular],
            "tennisSingleHeaderIcon": [Circular],
            "tennisUnForced": [Circular],
            "tennisWinner": [Circular],
            "themeGradientImage": [Circular],
            "threeDotIcon": [Circular],
            "thumbImage": [Circular],
            "timeKnobGray": [Circular],
            "timeKnobOrange": [Circular],
            "townsCupIcon": [Circular],
            "townsCupLogo": [Circular],
            "unCheckWhiteBorder": [Circular],
            "uncheckBox": [Circular],
            "uncheckEditor": [Circular],
            "uncheckWhite": [Circular],
            "undoImage": [Circular],
            "unlikeImage": [Circular],
            "unmute": [Circular],
            "upcomingMatch": [Circular],
            "usaImage": [Circular],
            "vertical3Dot": [Circular],
            "videoFullScreen": [Circular],
            "videoLoading": [Circular],
            "videoMuteSound": [Circular],
            "videoNormalScreen": [Circular],
            "videoPauseButton": [Circular],
            "videoPauseIcon": [Circular],
            "videoPlayBtn": [Circular],
            "videoPlayButton": [Circular],
            "videoPlayIcon": [Circular],
            "videoThumb": [Circular],
            "videoUnMuteSound": [Circular],
            "volleyballLoader": [Circular],
            "warningIcon": [Circular],
            "welcomeImage1": [Circular],
            "welcomeImage2": [Circular],
            "welcomeImage3": [Circular],
            "welcomeImage4": [Circular],
            "whiteTick": [Circular],
            "whiteTrophySmall": [Circular],
            "whiteUncheck": [Circular],
            "yellowCardImage": [Circular],
            "yellowCardSelected": [Circular],
            "yellowCheckBox": [Circular],
            "yellowDownArrow": [Circular],
            "yellowQBGroup": [Circular],
            "yellowQBUser": [Circular],
            "yellowRatingStar": [Circular],
            "youtubePlayButton": [Circular],
          },
        }
      }
      style={
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <RCTScrollView
      bounces={false}
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <View>
        <Text
          style={
            Object {
              "color": "#FFFFFF",
              "fontFamily": "Roboto-Bold",
              "fontSize": 25,
              "marginLeft": 49.5,
              "marginTop": 152,
              "textAlign": "left",
            }
          }
        >
          Log in
        </Text>
        <View
          style={
            Object {
              "marginTop": 30,
            }
          }
        >
          <View
            style={
              Array [
                Object {
                  "alignContent": "center",
                  "alignSelf": "stretch",
                  "backgroundColor": "#FCFCFC",
                  "borderRadius": 4,
                  "elevation": 1,
                  "flex": 1,
                  "flexDirection": "row",
                  "marginHorizontal": 15,
                  "shadowColor": "#A0A0A0",
                  "shadowOffset": Object {
                    "height": 1,
                    "width": 0,
                  },
                  "shadowOpacity": 0.3,
                  "shadowRadius": 1,
                },
                Object {
                  "alignSelf": "center",
                  "backgroundColor": "rgba(255,255,255,0.9)",
                  "shadowColor": "#616161",
                  "shadowOffset": Object {
                    "height": 4,
                    "width": 0,
                  },
                  "shadowOpacity": 0.5,
                  "shadowRadius": 4,
                  "width": 610,
                },
                Object {
                  "height": 40,
                },
              ]
            }
          >
            <TextInput
              autoCapitalize="none"
              autoCorrect={false}
              editable={true}
              keyboardType="email-address"
              multiline={false}
              onChangeText={[Function]}
              placeholder="Email"
              placeholderTextColor="#FF7F00"
              secureTextEntry={false}
              style={
                Array [
                  Object {
                    "color": "#333333",
                    "flex": 1,
                    "fontFamily": "Roboto-Regular",
                    "fontSize": 16,
                    "height": "100%",
                    "paddingHorizontal": 10,
                  },
                  undefined,
                  Object {
                    "height": 40,
                  },
                ]
              }
              testID="email-input"
              textAlignVertical="center"
              value="makani20@gmail.com"
            />
          </View>
          <View
            style={
              Object {
                "alignSelf": "center",
                "backgroundColor": "rgba(255,255,255,0.9)",
                "borderRadius": 5,
                "color": "black",
                "elevation": 3,
                "flexDirection": "row",
                "fontFamily": "Roboto-Regular",
                "fontSize": 16,
                "height": 40,
                "justifyContent": "space-between",
                "marginTop": 15,
                "paddingHorizontal": 10,
                "shadowColor": "#616161",
                "shadowOffset": Object {
                  "height": 4,
                  "width": 0,
                },
                "shadowOpacity": 0.5,
                "shadowRadius": 4,
                "width": 610,
              }
            }
          >
            <TextInput
              keyboardType="default"
              onChangeText={[Function]}
              placeholder="Password"
              placeholderTextColor="#FF8A01"
              secureTextEntry={true}
              style={
                Object {
                  "alignSelf": "center",
                  "borderRadius": 5,
                  "color": "black",
                  "fontFamily": "Roboto-Regular",
                  "fontSize": 16,
                  "height": 40,
                  "paddingHorizontal": 0,
                  "width": 487.5,
                }
              }
              testID="password-input"
              value="123456"
            />
            <View
              accessible={true}
              collapsable={false}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "justifyContent": "center",
                  "opacity": 1,
                }
              }
            >
              <Text
                style={
                  Object {
                    "color": "#FF7F00",
                    "fontSize": 10,
                    "textDecorationLine": "underline",
                  }
                }
              >
                SHOW
              </Text>
            </View>
          </View>
        </View>
        <View
          style={
            Object {
              "marginTop": 57.5,
            }
          }
        >
          <View
            accessible={true}
            collapsable={false}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignContent": "center",
                "backgroundColor": "#FFFFFF",
                "borderRadius": 40,
                "height": 45,
                "marginLeft": "7%",
                "marginRight": "5%",
                "marginTop": 0,
                "opacity": 1,
                "shadowColor": "#616161",
                "shadowOffset": Object {
                  "height": 2,
                  "width": 0,
                },
                "shadowOpacity": 0.5,
                "shadowRadius": 5,
                "width": "86%",
              }
            }
            testID="login-button"
          >
            <Text
              style={
                Array [
                  Object {
                    "fontFamily": "Roboto-Bold",
                    "fontSize": 16,
                    "height": 50,
                    "padding": 12,
                    "textAlign": "center",
                  },
                  Object {
                    "color": "#FF7F00",
                  },
                ]
              }
            >
              LOG IN
            </Text>
          </View>
          <View
            accessible={true}
            collapsable={false}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "opacity": 1,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#FFFFFF",
                  "fontFamily": "Roboto-Medium",
                  "fontSize": 14,
                  "marginTop": 41,
                  "textAlign": "center",
                }
              }
            >
              Forgot Password?
            </Text>
          </View>
        </View>
      </View>
    </RCTScrollView>
  </View>
  <RCTSafeAreaView
    emulateUnlessSupported={true}
  >
    <View
      style={
        Object {
          "bottom": 16,
        }
      }
    >
      <View
        accessible={true}
        collapsable={false}
        focusable={true}
        hitSlop={
          Object {
            "bottom": 15,
            "left": 15,
            "right": 15,
            "top": 15,
          }
        }
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignSelf": "center",
            "opacity": 1,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#FFFFFF",
              "fontFamily": "Roboto-Regular",
              "fontSize": 16,
              "textAlign": "center",
            }
          }
        >
          Not a member yet?
          <Text>
             
          </Text>
          <Text
            style={
              Object {
                "fontFamily": "Roboto-Bold",
                "textDecorationLine": "underline",
              }
            }
          >
            Sign Up
          </Text>
        </Text>
      </View>
    </View>
  </RCTSafeAreaView>
</BVLinearGradient>
`;
